com.playeveryware.eos/Runtime/Core/Utility/StorageUtility.cs

```diff
@@ -40,7 +51,7 @@ namespace PlayEveryWare.EpicOnlineServices.Utility
     /// <summary>
     /// Utility class used for a variety of File tasks.
     /// </summary>
-    public static class FileUtility
+    internal static class StorageUtility
```

Comment:
`StorageUtility` is kind of _icky_, I think we should go with `FileSystemUtility`.

Reply:
It has been so renamed.